### ISSUE TITLE
Ignore minor version if AutoMinorVersionUpgrade is set to true

### ISSUE THREAD

**Issue by vmusaa** [aws-cloudformation/cloudformation-coverage-roadmap#1309]
### Name of the resource

AWS::RDS::DBInstance

### Resource name

AWS::RDS::DBInstance

### Description

The AWS::RDS::DBInstance resource results in drift when the `AutoMinorVersionUpgrade`property is set to true and there is a minor version update. This causes the stack to go into drift. 

### Enhancement Request ask - 

Ignore these minor version upgrades while performing drift detection on a stack with AWS::RDS::DBInstance resource and `AutoMinorVersionUpgrade` set to True. 

### Document upgrade Request - https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/1308

**Comment by rittneje** [comment]
I don't think this should be excluded from drift detection. Part of the benefit of drift detection is knowing whether you diverged from your IaC.

What I think we need instead is a way not to specify the minor version at all.
